### PR TITLE
Add example that changes the pod network cidr of calico

### PR DESCRIPTION
**Pull request by angelbarrera92**
Hi Team!

I have added this example as i consider it relevant to be documented in the documentation site.

Feedback is welcomed

### CODE REVIEW

**Comment by nutellinoit** [comment]
LGTM

File: examples/custom-pod-network-cidr/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+metadata:
+  name: calico-node
+spec:
+  template:
+    spec:
+      containers:
+        - name: calico-node
+          env:
+          - name: CALICO_IPV4POOL_CIDR
+            value: "172.16.0.0/16" # PUT YOUR NETWORK CIDR HERE
```
Comment (lnovara) [inline examples/custom-pod-network-cidr/README.md]: Can we put another CIDR in the example? Just not to confuse users.

Comment (angelbarrera92) [reply examples/custom-pod-network-cidr/README.md]: sure!

Comment (ralgozino) [reply examples/custom-pod-network-cidr/README.md]: You've marked this as resolved, but I think you forgot to push the changes or maybe you changed only the patch and forgot to also change the README?

Comment (angelbarrera92) [reply examples/custom-pod-network-cidr/README.md]: My fault. Updated ;) Thanks!

**Pull request merged**